proposals/NNNN-test-scripts-common-modules-refactoring.md

```diff
@@ -0,0 +1,96 @@
+# Test Scripts - Refactoring of Common modules
+
+* Proposal: [SDL-NNNN](NNNN-test-scripts-common-modules-refactoring.md)
+* Author: [Dmytro Boltovskyi](https://github.com/dboltovskyi)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+ATF Test Scripts use a lot of commonly used functions. Such as:
+
+  - data accessors: file, database
+  - functions for tables: print, copy
+  - sequences:
+    - create mobile session
+    - register/activate mobile application
+    - perform policy table update
+  - etc.
+
+The purpose of this proposal is to clean up these common modules.
```

Comment:
@dboltovskyi ,
and "The purpose of this proposal is to clean up these common modules." better to move to solution